packages/cardpay-cli/README.md

```diff
@@ -6,27 +6,43 @@ CLI tool for basic actions in Cardpay
 
 # Commands
 - [Commands](#commands)
-  - [`yarn cardpay bridge <TOKEN_ADDRESS> <AMOUNT> --network=NETWORK [--mnemonic=MNEMONIC]`](#yarn-cardpay-bridge-token_address-amount---networknetwork---mnemonicmnemonic)
+  - [`yarn cardpay bridge <AMOUNT> <TOKEN_ADDRESS> --receiver=<RECEIVER> --network=NETWORK [--mnemonic=MNEMONIC]`](#yarn-cardpay-bridge-amount-token_address---receiverreceiver---networknetwork---mnemonicmnemonic)
+  - [`yarn cardpay await-bridged <FROM_BLOCK> [RECIPIENT] --network=NETWORK [--mnemonic=MNEMONIC]`](#yarn-cardpay-await-bridged-from_block-recipient---networknetwork---mnemonicmnemonic)
   - [`yarn cardpay prepaidcard-create <SAFE_ADDRESS> <TOKEN_ADDRESS> <amounts..> --network=NETWORK [--mnemonic=MNEMONIC]`](#yarn-cardpay-prepaidcard-create-safe_address-token_address-amounts---networknetwork---mnemonicmnemonic)
   - [`yarn cardpay safes-view [ADDRESS] --network=NETWORK [--mnemonic=MNEMONIC]`](#yarn-cardpay-safes-view-address---networknetwork---mnemonicmnemonic)
 
 
 
-## `yarn cardpay bridge <TOKEN_ADDRESS> <AMOUNT> --network=NETWORK [--mnemonic=MNEMONIC]`
+## `yarn cardpay bridge <AMOUNT> <TOKEN_ADDRESS> --receiver=RECEIVER --network=NETWORK [--mnemonic=MNEMONIC]`
 
 Bridge tokens from L1 wallet to L2 safe
 
 ```
 USAGE
-  $ yarn cardpay bridge <tokenAddress> <amount> --network=NETWORK [--mnemonic=MNEMONIC]
+  $ yarn cardpay bridge <amount> <tokenAddress> --receiver=RECEIVER --network=NETWORK [--mnemonic=MNEMONIC]
```

Comment:
I'd recommend a positional arg within a sub command like we are doing for `amount` and `tokenAddress`

Reply:
Done ✅